src/collection/layout.js

```diff
@@ -26,6 +42,11 @@ var elesfn = ({
           node.silentPosition( { x: 0, y: 0 } );
         }
 
+        if ( options.spacingFactor && options.spacingFactor !== 1){
+          var spacing = Math.abs(options.spacingFactor);
```

Comment:
Right now `var spacing = Math.abs(options.spacingFactor)` is outside the function so that it doesn't get called *n* times in the no-animation case. It could also be moved inside the function which would make everything more concise (and would be one less argument that needs to be passed).